source/agent/addons/videoSwitch/binding.gyp

```diff
@@ -0,0 +1,39 @@
+{
+  'targets': [{
+    'target_name': 'videoSwitch',
+    'sources': [
+      'addon.cc',
+      'VideoSwitchWrapper.cc',
+      '../../../core/owt_base/MediaFramePipeline.cpp',
+      '../../../core/owt_base/selector/VideoQualitySwitch.cpp',
+    ],
+    'include_dirs': [
+      "<!(node -e \"require('nan')\")",
+      '$(CORE_HOME)/common',
+      '$(CORE_HOME)/owt_base',
+      '$(CORE_HOME)/owt_base/selector',
+      '$(DEFAULT_DEPENDENCY_PATH)/include',
+      '$(CUSTOM_INCLUDE_PATH)'
+    ],
+    'libraries': [
+      '-lboost_system',
```

Comment:
Do not link to boost if you're not using it in this addon.

Reply:
The included header file "MediaFramePipeline.h" links to boost